archive/examples/prop_encodable.lean

```diff
@@ -0,0 +1,94 @@
+/-
+Copyright (c) 2019 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad
+
```

Comment:
We're not generating html output for the archive right now, but I think it would still be good to follow the standard doc conventions: everything below this line can move to a module doc `/-!` immediately after the imports.